README.md

```diff
@@ -264,6 +294,10 @@ In case you have choosen the STM Spirit1 Sub-1 GHz RF expansion board [X-NUCLEO-
 
 After changing the radio shield, you need to recompile the application.
 
+#### Wi-SUN RF shield
+
+Currently there is only one radio shield with a support for Wi-SUN, the STM S2LP radio shield.
```

Comment:
Should we add instruction because If I remember correctly some platform need jump wire before radio work.